src/main/java/org/spdx/utility/license/LicenseExpressionParser.java

```diff
@@ -455,8 +455,7 @@ private static LicenseAddition parseSimpleLicenseAdditionToken(String token, IMo
 	}
 
 	/**
-	 * Converts a string token into its equivalent license
-	 * checking for a listed license
+	 * Converts a string token into its equivalent license either a listed license or a LicenseRef-
```

Comment:
```suggestion
	 * Converts a string token into its equivalent license - either a listed license or a LicenseRef
```

Copied from `parseSimpleLicenseTokenCompatV2()`